coords/resolver.py

```diff
@@ -167,10 +174,154 @@ def identify_name(name):
             return None
 
     if any([o is None for o in out]):
-        raise IOError("Incomplete response from CADC name resolver: {0}".format(out))
+        raise IOError("Incomplete response from CADC name resolver: {}".format(out))
 
     out = tuple(out)
-    v5("Position = {0}".format(out))
+    v5("Position = {}".format(out))
     return out
 
+
+def identify_name_sesame(name):
+    """Find the coordinates of an Astronomical object from CDS/Sesame.
+
+    Use the Sesame name resolver to identify the given object name.
+
+    Parameters
+    ----------
+    name : str
+        The name of the object. This sent to the Sesame server.
+
+    Returns
+    -------
+    ra, dec, coordsys : number, number, string
+        The coordinates, in decimal degrees, and coordinate system
+        of the object if it was found. The coordsys value is
+        always set to 'ICRS'.
+
+    Raises
+    ------
+    ValueError
+        This is raised if the object is unknown
+    IOError
+        This is raised if there was an error contacting the CADC
+        service, or the return value could not be understood.
+
+    Notes
+    -----
+    Other errors from the Python networking stack may also be raised.
+
+    """
+
+    # Note: just want first response, search NED first, and select "XML" output.
+    #
+    tname = quote_plus(name)
+    url = 'https://cdsweb.u-strasbg.fr/cgi-bin/nph-sesame/-ox/~NSV?' + tname
+
+    # When there's no match the response document is different, rather than
+    # being reported by a HTTP error-response code.
+    #
+    try:
+        v5("Sesame name query: {0}".format(url))
+        rsp = urlopen(url)
+
+    except HTTPError as he:
+        code = he.getcode()
+        v5("Error from Sesame name resolver - status = {0}".format(code))
+        v5("HTTPError code={0}".format(code))
+        v5(str(he))
+        raise he
+
+    except URLError as ue:
+        # Is this a sufficient check?
+        v5("Error opening URL: {0}".format(ue))
+        v5("error.reason = {0}".format(ue.reason))
+        if ue.reason.errno == 8:
+            raise IOError("Unable to connect to the Sesame Name Resolver")
+
+        raise
```

Comment:
should this be

`raise ue` 

?

Reply:
I'm pretty sure that a bare `raise` just re-raises the exception, so no need to plumb the value through:

```
In [1]: try:
   ...:     raise ValueError("bob was here")
   ...: except Exception as exc:
   ...:     print("no no no")
   ...:     raise
   ...:
no no no
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-a0972d363b62> in <module>
      1 try:
----> 2     raise ValueError("bob was here")
      3 except Exception as exc:
      4     print("no no no")
      5     raise

ValueError: bob was here
```